api/src/test/java/org/openmrs/module/fhir2/api/translators/impl/DiagnosticReportTranslatorImplTest.java

```diff
@@ -0,0 +1,190 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.module.fhir2.api.translators.impl;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.contains;
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.hasProperty;
+import static org.hamcrest.Matchers.notNullValue;
+import static org.mockito.Mockito.when;
+
+import java.util.Date;
+import java.util.List;
+
+import org.hl7.fhir.r4.model.CodeableConcept;
+import org.hl7.fhir.r4.model.Coding;
+import org.hl7.fhir.r4.model.DiagnosticReport;
+import org.hl7.fhir.r4.model.Reference;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import org.openmrs.Concept;
+import org.openmrs.Encounter;
+import org.openmrs.Obs;
+import org.openmrs.Patient;
+import org.openmrs.module.fhir2.api.translators.ConceptTranslator;
+import org.openmrs.module.fhir2.api.translators.EncounterReferenceTranslator;
+import org.openmrs.module.fhir2.api.translators.ObservationReferenceTranslator;
+import org.openmrs.module.fhir2.api.translators.PatientReferenceTranslator;
+
+public class DiagnosticReportTranslatorImplTest {
+	
+	private static final String PARENT_UUID = "249b9094-b812-4b0c-a204-0052a05c657f";
+	
+	private static final String CHILD_UUID = "faf75a02-5083-454c-a4ec-9a4babf26558";
+	
+	private static final String PATIENT_UUID = "249b9094-5083-454c-a4ec-9a4babf26558";
+	
+	@Mock
+	private EncounterReferenceTranslator encounterReferenceTranslator;
+	
+	@Mock
+	private PatientReferenceTranslator patientReferenceTranslator;
+	
+	@Mock
+	private ConceptTranslator conceptTranslator;
+	
+	@Mock
+	private ObservationReferenceTranslator observationReferenceTranslator;
+	
+	DiagnosticReportTranslatorImpl translator;
+	
+	Obs obsGroup;
+	
+	@Before
+	public void setup() {
+		MockitoAnnotations.initMocks(this);
```

Comment:
Instead of this line, you could just annotate the class as `@RunWith(MockitoJUnitRunner.class)`